devices/pine64-pinephone/u-boot/default.nix

```diff
@@ -58,15 +86,21 @@ in
     CONFIG_REGEX=y
   '';
 }).overrideAttrs(old: rec {
-  version = "2020.07";
+  version = "2021.01";
   src = fetchurl {
     url = "ftp://ftp.denx.de/pub/u-boot/u-boot-${version}.tar.bz2";
-    sha256 = "0sjzy262x93aaqd6z24ziaq19xjjjk5f577ivf768vmvwsgbzxf1";
+    sha256 = "0m04glv9kn3bhs62sn675w60wkrl4m3a4hnbnnw67s3l198y21xl";
   };
   postInstall = ''
     cp .config $out/build.config
   '';
   postPatch = old.postPatch + ''
     cat $extraConfigPath >> configs/$defconfig
   '';
+  nativeBuildInputs = [
+    (buildPackages.python3.withPackages(p: [
+      p.libfdt
+      p.setuptools # for pkg_resources
+    ]))
```

Comment:
This prefixes the existing python in the nativeBuildInputs; which in turn ensures this builds while the Nixpkgs upgrade to U-Boot is not yet in stable.